python-threatexchange/threatexchange/extensions/vpdq/vpdq_util.py

```diff
@@ -103,12 +146,12 @@ def read_file_to_hash(
         VPDQ features from the hash file"""
 
     with open(input_hash_filename, "r") as file:
-        return json_to_vpdq(file.read())
+        return OLD_json_to_vpdq(file.read())
 
 
-def dump_hash_to_file(
+def OLD_dump_hash_to_file(
```

Comment:
Ugh, all the old methods are still here as well. I'll kill them in a followup diff. Coding late at night = bad code -_-